setup.cfg

```diff
@@ -0,0 +1,21 @@
+[metadata]
+name = cockpit
+version = 0
+
+[options]
+packages = find:
+package_dir =
+    =src
```

Comment:
That looks really strange -- I understand that there can be a newline and indentation after `=` to have a multi-line value, but the repeated `=` looks like an error?

Reply:
This is copied verbatim from the setuptools documentation for src-layout packages: https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

From what I understand, the `package_dir=` attribute should be read as a multi-line mapping.  In this case our mapping contains a single item which maps the empty string to `src`.

The syntax is absolutely bizarre, but this is stock usage, so I don't think we need to document it.

Reply:
Thanks for the heads-up!